flang/lib/Optimizer/Dialect/FIROps.cpp

```diff
@@ -2243,7 +2244,7 @@ static constexpr llvm::StringRef getTargetOffsetAttr() {
 template <typename A, typename... AdditionalArgs>
 static A getSubOperands(unsigned pos, A allArgs,
                         mlir::DenseIntElementsAttr ranges,
-                        AdditionalArgs &&...additionalArgs) {
+                        AdditionalArgs &&... additionalArgs) {
```

Comment:
clang-format objects to the space after the `...`.

Reply:
I'll double check that I'm using the latest clang-format. (They differ on this.)